docs/ff-concepts/file-handling/displaying-media.md

```diff
@@ -0,0 +1,292 @@
+### Customization Options
+
+- **Title:** Specify the audio title in the **Title** property. You can set this directly or bind it to a variable, such as an app state variable, API response, or Firestore document.
+- **Pause on Forward Navigation:** By default, the audio stops when navigating to another page.
+- **Play in Background:** Define how the audio behaves when the app moves to the background:
+    - **Enabled:** The audio continues to play.
+    - **Disabled, restore on foreground:** The audio pauses and resumes when the app becomes active again.
+    - **Disabled, pause:** The audio stops immediately when the app goes into the background.
+- **Colors:**
+    - **Background Color:** Customize the background using the **Fill Color** property.
+    - **Playback Button Color:** Adjust the colors of the play and pause buttons.
+    - **Active/Inactive Track Color:** Change the progress bar color that indicates the current playback position.
+- **Elevation:** Use the **Elevation** property to modify the shadow beneath the audio tile. A higher value increases the shadow size, while setting it to 0 removes the shadow.
+- **Text Styling:**
+    - **Title Text:** Personalize the title’s font, size, and color in the **Title Text Style** section.
+    - **Playback Duration Text:** Adjust the style of the playback duration text in the **Playback Duration Text Style** section.
+
+## Audio Recording
+
+You can implement audio recording functionality using the **Start Audio Recording** and **Stop Audio Recording** actions.
+
+:::warning
+
+Currently, audio recording is not supported in **Run** or **Test** modes due to certain limitations.
+
+:::
+
+### Start Audio Recording [Action]
+
+This action starts the recording. It also provides a name to the recording, which you can use later to stop the recording using the [Stop Audio Recording](#stop-audio-recording-action) *action.*
+
+Before adding this action, ensure you [request microphone permission](../../resources/projects/settings/project-setup.md#request-permission-action). Then, inside the **TRUE** section of the permission check, add the **Start Audio Recording** action. By default, the **Name** is a randomly generated string. You can change it to a more descriptive name for easier identification.
+
+:::tip
+
+After starting recording, you might want to update the state variables to reflect changes on the UI. For instance, you can enable/disable buttons or start recording animations to provide a visual cue of the ongoing process. This step allows you to enhance the user experience and provide real-time feedback during the recording.
+
+:::
+
+![start-audio-recording.avif](imgs/start-audio-recording.avif)
+
+
+### Stop Audio Recording [Action]
+
+This halts the ongoing recording using the name from the [Start Audio Recording](#start-audio-recording-action) action. To capture and play the recorded audio, make sure to specify the *Action Output Variable Name*, which can be used with the audio player.
```

Comment:
```suggestion
If you have multiple audio recording actions, all the Recorder object names (either auto-generated by FlutterFlow or manually set by the user) are listed under the Recorder Name dropdown. Choose the recorder object you want to stop, and it will stop the ongoing recording.

 To capture and play the recorded audio, make sure to specify the *Action Output Variable Name*, which can be used with the audio player.
```